src/gamelog.h

```diff
@@ -80,7 +80,7 @@ class Gamelog {
 	void GRFAddList(const GRFConfigList &newg);
 	void GRFRemove(uint32_t grfid);
 	void GRFAdd(const GRFConfig &newg);
-	void GRFBug(uint32_t grfid, uint8_t bug, uint64_t data);
+	void GRFBug(uint32_t grfid, ::GRFBug bug, uint64_t data);
```

Comment:
I like how it's not confusing :)

Reply:
Yes, that's an annoying naming conflict.